resqpy/grid/create_grid_xml.py

```diff
@@ -0,0 +1,379 @@
+"""Submodule containing functions to create a grid xml file."""
+
+import logging
+
+log = logging.getLogger(__name__)
+
+import resqpy.olio.xml_et as rqet
+import resqpy.property as rprop
+from resqpy.olio.xml_namespaces import curly_namespace as ns
+
+always_write_pillar_geometry_is_defined_array = False
+always_write_cell_geometry_is_defined_array = False
+
+
+def create_grid_xml(grid,
```

Comment:
Personally I like Connor's suggestion of hinting at privacy for these modules and/or functions with an underscore, assuming that this function is called from a wrapper Grid.create_xml() method.